### ISSUE TITLE
Add argument "namedetails" to Nominatim/OpenMapQuest

### ISSUE THREAD

**Issue by enriqueav** [geopy/geopy#367]
Hi.

When geocoding using Nominatim, similar to the parameter already supported `addressdetails `, there is another one that is called `namedetails`:
https://nominatim.org/release-docs/develop/api/Search/#output-details

It basically adds all the alternative names of the place:
```
	"namedetails": {
		"name": "京都市",
		"name:cs": "Kjóto",
		"name:de": "Kyōto",
		...
		...
		"name:ja_kana": "きょうとし"
	}
```
I could work on this.

**Comment by enriqueav** [comment]
I went ahead and create a PR #368

**Comment by KostyaEsmukov** [comment]
Resolved by #368
